disease/etl/mondo.py

```diff
@@ -0,0 +1,214 @@
+"""Module to load disease data from Mondo Disease Ontology."""
+from .base import Base
+import logging
+from disease import PROJECT_ROOT
+from disease.database import Database
+from disease.schemas import Meta, SourceName, NamespacePrefix, Disease
+from pathlib import Path
+import requests
+from typing import Set, Dict
+import owlready2 as owl
+from rdflib.term import URIRef
+
+
+logger = logging.getLogger('disease')
+logger.setLevel(logging.DEBUG)
+
+
+MONDO_PREFIX_LOOKUP = {
+    "NCIT": NamespacePrefix.NCIT,
+    "DOID": NamespacePrefix.DO,
+    "SCTID": NamespacePrefix.SNOMEDCT,
```

Comment:
I don't think we should be using snomedct due to their licensing issues we faced in therapy normalization. I might be wrong, but we should double check the license terms for this and other xrefs.

Reply:
checks out. We will have to make do with the 200 other xrefs provided by Mondo